test/vectors/issuance/2.issue-credential_offer.json

```diff
@@ -0,0 +1,198 @@
+                                 ],
+                                 "schema":{
+                                    "type":"string"
+                                 },
+                                 "label":"Given Name"
+                              },
+                              {
+                                 "path":[
+                                    "$.credentialSubject.familyName"
+                                 ],
+                                 "schema":{
+                                    "type":"string"
+                                 },
+                                 "label":"Family Name"
+                              },
+                              {
+                                 "path":[
+                                    "$.credentialSubject.gender"
+                                 ],
+                                 "schema":{
+                                    "type":"string"
+                                 },
+                                 "fallback":"Not disclosed",
+                                 "label":"Gender"
+                              },
+                              {
+                                 "path":[
+                                    "$.credentialSubject.birthDate"
+                                 ],
+                                 "schema":{
+                                    "type":"string",
+                                    "format":"date"
+                                 },
+                                 "label":"Date of Birth"
+                              },
+                              {
+                                 "path":[
+                                    "$.credentialSubject.birthCountry"
+                                 ],
+                                 "schema":{
+                                    "type":"string"
+                                 },
+                                 "label":"Country of Birth"
+                              },
+                              {
+                                 "path":[
+                                    "$.credentialSubject.residentSince"
+                                 ],
+                                 "schema":{
+                                    "type":"string",
+                                    "format":"date"
+                                 },
+                                 "label":"Resident Since"
+                              }
+                           ]
+                        },
+                        "styles":{
+                           "thumbnail":{
+                              "uri":"https://file-server.trustbloc.local:12096/images/credential--uscis-icon.svg",
+                              "alt":"Citizenship & Immigration Services"
+                           },
+                           "hero":{
+                              "uri":"https://example.com/trust.png",
```

Comment:
consider a data uri instead, that will actually render

Reply:
done